addfunction.js

```diff
@@ -74,281 +74,339 @@ let a3 = document.createElement('a');
+    button11, div18, button12, img21, p21, div19, button13, div20,
+    button14);
+  div14.append(h34, div15);
+  div16.append(button10, div17, button11, div18, button12);
+  // img21.append(p21)
+  div19.append(button13, div20, button14);
+  section2.setAttribute('class', 'section_pop_window');
+  div14.setAttribute('class', 'pop_container pop_window');
+  h34.setAttribute('class', 'pop_title');
+  div15.setAttribute('class', 'closeicon');
+  div15.setAttribute('onclick', 'section2.remove()');
+  div16.setAttribute('class', 'pop_container move_buttons');
+  button10.setAttribute('class', 'pop_button_edit');
+  button10.setAttribute('type', 'button');
+  button11.setAttribute('class', 'pop_button_edit');
+  button11.setAttribute('type', 'button');
+  button12.setAttribute('class', 'pop_button_edit');
+  button12.setAttribute('type', 'button');
+  div17.setAttribute('class', 'divider');
+  div18.setAttribute('class', 'divider');
+  img21.setAttribute('class', 'portfolio_image');
+  img21.setAttribute('src', './images/projectpage_pop.svg');
+  img21.setAttribute('alt', 'portfolio_pic');
+  p21.setAttribute('class', 'pop_content');
+  div19.setAttribute('class', 'buttons_live_pop');
+  button13.setAttribute('class', 'padding_live live_link1');
+  button13.setAttribute('onclick', "window.location.href='https://github.com/Medini-Rajendra/Mobile_Webpage_Menu';");
+  div20.setAttribute('class', 'divider_live');
+  button14.setAttribute('class', 'padding_live live_link2');
+  button14.setAttribute('onclick', "window.location.href='https://github.com/Medini-Rajendra/Mobile_Webpage_Menu';");
+
+  h34.textContent = 'Project name goes here';
+  div15.textContent = '×';
+  button10.textContent = 'HTML/CSS';
+  button11.textContent = 'Ruby on Rails';
+  button12.textContent = 'Javascript';
+  p21.textContent = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi <br><br>Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.';
+  button13.textContent = 'See live';
+  button14.textContent = 'See source';
+}
+
+const form = document.querySelector('form');
+
+function validation() {
+  const mail = document.getElementById('mail').value;
+  const mail1 = document.getElementById('mail_grid').value;
+
+  const error = document.getElementById('error');
+  const error1 = document.getElementById('error_grid');
+  const pattern = /[A-Z]/;
+
+  if (mail.match(pattern) || mail1.match(pattern)) {
+    error.innerText = 'email must be in lower case';
+    error1.innerText = 'email must be in lower case';
+  } else {
+    error.innerText = 'you\'re cool';
+    error1.innerText = 'you\'re cool';
+  }
```

Comment:
**Change Required**
- [ ] Good effort validating email as lowercase. However, the error messages should be moved out of the input field and placed next to the submit button just above it. This is in line with the `figma design specification`.

Reply:
I have made the necessary changes for this issue. It's fixed now as per the suggestion.